owrrm/src/main/java/com/facebook/openwifi/rrm/RRMAlgorithm.java

```diff
@@ -183,7 +195,7 @@ public AlgorithmResult run(
 			ChannelOptimizer optimizer;
 			switch (mode) {
 			default:
-				if (!allowDefaultMode || !mode.isEmpty()) {
```

Comment:
this was correct before, why did you change it? (error if either default/empty is forbidden OR invalid non-empty mode name was passed)

Reply:
I misunderstood something, let me change it back.